tests/test_models/test_position_encoding.py

```diff
@@ -0,0 +1,37 @@
+import pytest
+import torch
+
+from mmdet.models.utils import LearnedPositionEmbedding, SinePositionEmbedding
+
+
+def test_position_encoding_sine(num_pos_feats=16, batch_size=2):
+    with pytest.raises(AssertionError):
```

Comment:
What assertion is tested?

Reply:
The type of scale which is not in [int,float] will raise assertion error.

Reply:
Please add this as a comment in the code.